app/forms/declaration/DepartureTransport.scala

```diff
@@ -23,10 +23,19 @@ import play.api.data.Forms.{mapping, optional, text}
 import play.api.libs.json.Json
 import utils.validators.forms.FieldValidator.{isContainedIn, noLongerThan, _}
 
-case class DepartureTransport(borderModeOfTransportCode: String, meansOfTransportOnDepartureType: String, meansOfTransportOnDepartureIDNumber: String)
+case class DepartureTransport(
+  borderModeOfTransportCode: String,
+  meansOfTransportOnDepartureType: String,
+  meansOfTransportOnDepartureIDNumber: String
+) {
+
+  val extractModeValue: String = TransportCodes.extractModeOfTransportValue(borderModeOfTransportCode)
+  val extractTypeMode: String = TransportCodes.extractBorderTransportValue(meansOfTransportOnDepartureType)
+
+  def transportDetails: Seq[String] = Seq(extractTypeMode, meansOfTransportOnDepartureIDNumber)
```

Comment:
Once again, seems a little strange to return a `Seq[String]`

Reply:
Removed